src/auth.ts

```diff
@@ -456,6 +456,7 @@ const getAuth = function (
 			resource: 'user',
 			id,
 			body: {
+				// @ts-expect-error this actually exists on the model but it is only on the update allow list
```

Comment:
Is that v right?
```suggestion
				// @ts-expect-error this actually exists but isn't part of the generated model but it is only on the update allow list and the allowlist atm needs it to also be in the Read as well
```
I would argue that we should augment the Write allowlist if it's easy, to avoid the confusion in case any consumer tries to do the same request manually.

Reply:
@Page- is against touching the allowlist iirc but I would prefer to defer this to afterwards

Reply:
Yes, I think having something you can write but not read is a real anti-pattern and I think the correct answer is they should not exist, since in practice they're actions and not really writes to the model (they might trigger writes to the model as part of executing the action, but the action is the desired outcome because you can't observe the write itself..)